dash/backend/src/modules/reports/dao/reports.dao.ts

```diff
@@ -254,13 +265,37 @@ export class ReportsDao {
             query.limit(options.limit);
         }
 
-        const queryResults = await query.then((response) => {
-            return plainToClass(ReportsRunningVulnerabilitiesDto, response);
+        const queryResults: ReportsRunningVulnerabilitiesDto[] = await query.then((response) => {
+          // plainToInstance returns a single even when it's an array --> proper typing requires longer form
```

Comment:
This comment is inaccurate, and you shouldn't need to be doing this mapping here. The functions js doc is: 'Converts plain (literal) object to class (constructor) object. Also works with arrays'.

plainToInstance(ReportsRunningVulnerabilitiesDto, response) will return a single object if response is a single object, or process & return the entire array if response is an array.

If your IDE isn't liking this, then it is likely configured with an incorrect linteror typescript config, or you need to give response a type since right now it is defaulting to any, it should probably be any[])